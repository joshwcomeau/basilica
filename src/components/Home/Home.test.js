/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { Home } from './index';

describe('Home', () => {
  // TODO: wrap in <provider> with a test store
  xit('renders correctly', () => {
    const tree = renderer.create(
      <Home
        city="montreal"
        initializeShopify={() => {}}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
