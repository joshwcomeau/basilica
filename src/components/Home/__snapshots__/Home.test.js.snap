exports[`Home renders correctly 1`] = `
<div
  className="home">
  <div
    className="hero"
    style={
      Object {
        "backgroundImage": "url(test-file-stub)"
      }
    }>
    <div
      className="sticky-logo is-not-stuck"
      style={undefined}>
      <h2
        className="logo">
        Basilica
      </h2>
    </div>
  </div>
  <div
    className="header-contents">
    City: Montreal
  </div>
  <div
    className="header-backdrop" />
  <div
    className="editorial">
    Words and Stuff
  </div>
  <div
    className="map">
    <div
      className="zoom-control">
      <button
        onClick={undefined}>
        +
      </button>
      <button
        onClick={false}>
        -
      </button>
    </div>
  </div>
  <div
    className="products-list">
    Your Component Here :)
  </div>
</div>
`;
